pkgs/by-name/dn/dnsmap/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  autoconf,
+  automake,
+}:
```

```suggestion
{
  autoconf,
  automake,
  fetchFromGitHub,
  lib,
  stdenv,
}:
```
There isn't consensus on this, so this is an optional comment.
I favor alphabetical order because it is very objective.